### DISCUSSION TITLE
What timezone formats are supported?

### DISCUSSION THREAD

**Discussion by punksinatra** [datenstrom/community#269] [Ask a question] [answered]
Hi noob question. I know where to set the timezone but I am not sure what format does it accept? 

I need to change the timezone to GMT +8 to match my local timezone. Thanks

**Comment by schulle4u** [comment, marked as answer]
Yellow uses the standard timezone settings of php. A list of timezones can be found here: 

http://php.net/manual/en/timezones.php

Just pick the city that best suits your timezone and copy the value into your `system/config/config.ini`. 

HTH
Steffen

**Comment by punksinatra** [comment]
Hey that worked. Thanks

**Comment by xsinger** [comment]
`system/config/config.ini`

```
Timezone: UTC +8
```
Is OK

**Comment by schulle4u** [comment]
[But not recommended](http://php.net/manual/en/timezones.others.php)

> Please do not use any of the timezones listed here (besides UTC), they only exist for backward compatible reasons, and may expose erroneous behavior.
